umeå/meetup.yaml

```diff
@@ -0,0 +1,86 @@
+city: Umeå
+country: sweden
+meetupID: Cloud-Native-Northern-Sweden
+meetups:
+- address: Tvistevägen 47
+  attendees: 19
+  date: "2019-05-28T17:00:00Z"
+  duration: 2h0m0s
+  id: 261450080
+  name: 'Trends in cloud native - a summary from KubeCon + CloudNativeCon Barcelona '
+  presentations:
+  - endTime: "18:30"
+    slides: ""
+    speakers:
+    - lindaalmersson
+    - tordsson
+    startTime: "17:00"
+    title: ""
+  sponsors:
+    other: null
+    venue: elastisys
+- address: Västra Norrlandsgatan 10D
+  attendees: 36
+  date: "2019-05-07T17:00:00Z"
+  duration: 2h0m0s
+  id: 260276456
+  name: What does “production ready” really mean for a Kubernetes cluster?
+  presentations:
+  - endTime: "18:30"
+    slides: https://speakerdeck.com/luxas/what-does-production-ready-really-mean-for-a-kubernetes-cluster-umea-may-2019
+    speakers:
+    - luxas
+    startTime: "17:30"
+    title: What does “production ready” really mean for a Kubernetes cluster?
+  sponsors:
+    other: null
+    venue: xlent
+- address: Södra Ersmarksgatan 7
+  attendees: 45
+  date: "2019-02-28T17:00:00Z"
+  duration: 2h0m0s
+  id: 257861742
+  name: Cloud native lightning talks
+  presentations:
+  - endTime: "18:30"
+    slides: ""
+    speakers:
+    - eriklindskog
+    - lindaalmersson
+    - tordsson
+    - jonasengman
+    - magnusforsell
+    - anderssigfridsson
+    - kimnilsson
+    - martinnordin
+    - jonaseklund
+    - stevewidinghoff
+    - martonszucs
+    - perengman
+    startTime: "17:30"
+    title: Lightning talks
+  sponsors:
+    other: null
+    venue: codemill
+- address: Tvistevägen 47
+  attendees: 45
+  date: "2018-11-20T17:00:00Z"
+  duration: 2h0m0s
+  id: 255929396
+  name: Cloud-native, Kubernetes and trends
+  presentations:
+  - endTime: "18:30"
+    slides: https://speakerdeck.com/luxas/intro-to-the-cloud-native-world-of-kubernetes-november-2018
+    speakers:
+    - luxas
+    startTime: "17:30"
+    title: Intro to the cloud-native world with Kubernetes
+  sponsors:
+    other: null
+    venue: elastisys
+members: 128
```

Comment:
this part has been dropped from master, sorry. remove it and re-run the generator